content/en/docs/reference/rules/supported-fields/index.md

```diff
@@ -205,14 +212,8 @@ Name | Type | Description
 `container.healthcheck` | CHARBUF | The container's health check. Will be the null value ("N/A") if no healthcheck configured, "NONE" if configured but explicitly not created, and the healthcheck command line otherwise
 `container.liveness_probe` | CHARBUF | The container's liveness probe. Will be the null value ("N/A") if no liveness probe configured, the liveness probe command line otherwise
 `container.readiness_probe` | CHARBUF | The container's readiness probe. Will be the null value ("N/A") if no readiness probe configured, the readiness probe command line otherwise
-
-{{% alert color="primary" %}}
-
-The [k8s set of fields](#field-class-k8s) `k8s.ns.name` and `k8s.pod.*` (i.e., `k8s.pod.name`, `k8s.pod.id`, `k8s.pod.labels`, and `k8s.pod.label.*`) are populated with data fetched from the **container runtime**.
-
-Therefore, they can also be accessed without having the **Kubernetes Metadata Enrichment** functionality enabled (`-k` Falco option).
-
-{{% /alert %}}
```

Comment:
note for the reviewer: this is also replicated further down. that text is a bit risky since these pages are auto-generated and it's possible that it could be removed by accident, perhaps we could add it to Falco itself, it won't have the alert box but would be preserved more easily.